usmqe/web/tendrl/mainpage/tasks/models.py

```diff
@@ -0,0 +1,116 @@
+"""
+Common page model for tasks.
+"""
+
+
+from webstr.core import By, WebstrModel, PageElement, BaseWebElementHelper
+import webstr.patternfly.contentviews.models as contentviews
+import webstr.common.form.models as form
+
+LOCATION = "#/admin/tasks"
+
+
+class StatusIconHelper(BaseWebElementHelper):
+    """
+    StatusIcon helper (Selenium webelement wrapper).
+    Provides basic methods for manipulation with a task status icon.
+    """
+
+    @property
+    def value(self):
+        """
+        find status
+
+        Returns:
+            state of task
+        """
+        icon_class = self.get_attribute('class')
+        if 'pficon-error-circle-o' in icon_class:
+            return 'failed'
+        elif 'pficon-ok' in icon_class:
+            return 'finished'
+        elif 'pficon-warning-triangle-o' in icon_class:
+            return 'warning'
+        elif 'fa-spinner' in icon_class:
+            return 'new or processing'
+        else:
+            raise Exception('Unknown icon state')
```

Comment:
Is meaning of patternfly icons defined for Tendrl somewhere in design or other document? If yes, it would be useful to link it in the docstring or comment.